src/Avalonia.Controls/Shapes/Arc.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using Avalonia.Media;
+
+namespace Avalonia.Controls.Shapes
+{
+    public class Arc : Shape
```

Comment:
Looks good to me, but does it make sense to be moved to separated PR?

Reply:
@maxkatz6 https://github.com/AvaloniaUI/Avalonia/pull/6010